operations/toc-chair.md

```diff
@@ -0,0 +1,41 @@
+# TOC Chair
+
+This document serves to provide the TOC Chair with details on their responsibilities, role, and expectations that are in addition to those of a TOC member.
+
+## Chair selection and time committment
+
+The TOC Chair is an member of the TOC that is chosen by each seated TOC to lead and resent the TOC in the execution of it's work. The expected time commitment of a TOC member who is serving as Chair is roughly **20-25% of their monthly regular work time**, and may be higher or lower depending on existing community and project challenges, activity of the TOC, and other occurrences that may occupy their time (such as serving in an interim capacity on other groups). The Chair should also anticipate travel to conferences and offsites at least twice a year, with participation and attendence at various meetings during the CNCF flagship conference: KubeCon+CloudNativeCon (KCCN).
```

Comment:
> The Chair should also anticipate travel to conferences and offsites at least twice a year, with participation and attendence at various meetings during the CNCF flagship conference: KubeCon+CloudNativeCon (KCCN).

could be cut, as it is also mentioned as part of the expectations of the TOC members & in the task list below